fixtures/pages-workerjs-app/tests/index.test.ts

```diff
@@ -1,13 +1,58 @@
-import { execSync } from "child_process";
-import path from "path";
+import { execSync, fork } from "node:child_process";
+import path from "node:path";
+import type { Serializable } from "node:child_process";
 
 describe("Pages _worker.js", () => {
 	it.concurrent(
 		"should throw an error when the _worker.js file imports something",
 		() => {
 			expect(() =>
-				execSync("npm run dev", { cwd: path.resolve(__dirname, "..") })
+				execSync("npm run dev", {
+					cwd: path.resolve(__dirname, ".."),
+					stdio: "ignore",
+				})
 			).toThrowError();
 		}
 	);
+
+	it.concurrent(
```

Comment:
Would be great to actually validate that some bundled value is making it into the script e.g.

```ts
import { helloWorldString } from './otherFile'

export default {
  fetch() {
    return new Response(x)
  }
}
```

And the test:

```ts
expect(await fetch(`http://${ip}:${port}/`).then(resp => resp.text())).toContain("Hello, world!")
```

Reply:
Done